.travis.yml

```diff
@@ -1,8 +1,8 @@
-sudo: required
-language: bash
-
-services:
-  - docker
+language: go
```

Comment:
Do we need this for codecov? One of the reasons I didn't want to tag this as a "go" project is because at some point I'm hoping to have the CF ruby code move into here too - which means we're not just go.